operator/src/main/java/oracle/kubernetes/weblogic/domain/model/OnlineUpdate.java

```diff
@@ -0,0 +1,234 @@
+// Copyright (c) 2020, 2021, Oracle and/or its affiliates.
```

Comment:
@rjeberhard  Please review all of the Description text in this file (Johnny and I have already word-smithed).  We can then pass to Rosemary...